org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "opent2t-translator-com-insteon-binaryswitch",
+  "version": "0.2.0",
+  "description": "Insteon Binary Switch",
+  "homepage": "http://opentranslatorstothings.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/opent2t/translators.git"
+  },
+  "main": "thingTranslator.js",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava tests/test.js",
+    "unittest": "ava tests/unitTests.js"
+  },
+  "ava": {
+    "files": [
+      "tests/*.js"
+    ]
+  },
+  "dependencies": {
+    "opent2t": "^1.0.2",
+    "opent2t-translator-com-insteon-hub": "^0.2.0"
+  },
+  "devDependencies": {
+    "ava": "^0.15.2",
+    "es6-sleep": "*",
+    "opent2t-device-insteonhub": "file:../../../org.opent2t.sample.hub.superpopular/com.insteon.hub/test",
```

Comment:
I just followed the latest test files changes from [https://github.com/openT2T/translators/pull/135/commits](https://github.com/openT2T/translators/pull/135/commits). @ChueckFerring for comments.

---
In reply to: [98083860](https://github.com/openT2T/translators/pull/149#discussion_r98083860) [](ancestors = 98083860)